src/library/importer.py

```diff
@@ -17,14 +24,54 @@ def import_quests_to(*, destination_schema, quest_import_ids):
 
     dry_run = False
     with schema_context(destination_schema):
-        res = QuestResource().import_data(export_data, dry_run=dry_run)
+        # Explicitly import the campaign as well
+        result = QuestResource().import_data(export_data, dry_run=dry_run, import_campaign=True)
 
         object_ids = []
-        for row in res.rows:
+        for row in result.rows:
             if row.new_record:
                 object_ids.append(row.object_id)
-        # Set visible_to_students to False for imported quests
-        # since the imported quests will be orphans
+        # Set visible_to_students (published) to False for imported quests,
+        # using update() for efficiency and to avoid triggering full model validation or signals,
+        # since we're only flipping a simple boolean flag on multiple objects at once.
```

Comment:
This type of comment explaining WHY is super valuable.  What is usually obvious from reading the code, but WHY is not!